docs/integrations/graphql-api.md

```diff
@@ -63,7 +63,7 @@ In addition, filtering can be done on list of related objects as shown in the fo
   device_list {
     id
     name
-    interfaces(enabled: true) {
+    interfaces(filters: {enabled: true}) {
```

Comment:
This doesn't appear to work (with `strawberry-graphql-django` v0.34.0). I get the following error:

> Unknown argument 'filters' on field 'DeviceType.interfaces'.

Reply:
Oh, does this depend on #15557?

Reply:
Yes - it depends on that being merged.